app/views/journals/show.html.slim

```diff
@@ -3,13 +3,13 @@ h1= t_title(:show, Journal)
 .row
   .col-xs-12
     p
-      strong= "#{t_attr(:subject, Journal)}: "
```

Comment:
Try `/clients/816/journals/49` and replace the clientid 816 and journalid 49 to valid ones on your system. Run in german. Here it is not working